### PR TITLE
The Crate Animatening, A.K.A. I hate Crate Trigonometry

### PR DESCRIPTION
**Pull request by alsoandanswer**
- rscadd: "Adds cool animations to opening and closing crates."
  - soundadd: "Adds new sounds for opening and closing some cabinets and crates."
  - imageadd: "Resprites crates to look more swanky."
  - refactor: "Loot crates (A.K.A. Unusual Containers) should now spawn with more unique sprites."
  - rscadd: "Adds animation to opening and closing lockers and body bags."
  - refactor: "Refactors Crate and Locker code to be more internally consistent."
  - rscdel: "Removed Electropack Crate rigging mechanic for being unused and eating cable coils." 
dreamseeker_Wafv5IkdP7

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]
Has been testmerged for a few weeks now, works fine.

**Review by JohnWildkins — CHANGES_REQUESTED** [review]
see unit tests

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -391,38 +344,18 @@
 	name = "wall locker"
```
Comment (JohnWildkins) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:344]: no animation

**Review by JohnWildkins — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 2]
```diff
@@ -529,70 +373,85 @@
...
-	icon_closed = "largebins"
-	redlight = "largebinr"
-	greenlight = "largebing"
-	sparks = "largebinsparks"
-	emag = "largebinemag"
+	icon_door_overlay = "largebin"
+	icon_door_override = TRUE
+	icon_door = "largebin"
 
 /obj/structure/closet/crate/large
```
[the remark this answers is not in the document]
Comment (alsoandanswer) [reply code/game/objects/structures/crates_lockers/crates.dm:449]: intentional

anything with door_anim_time = 0 has no animation

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**